tests/snmp_fixtures/general-bgp-external-reset.snmprec

```diff
@@ -0,0 +1,9 @@
+1.3.6.1.2.1.1.3.0|67|1000000
+1.3.6.1.2.1.15.2.0|2|10
+1.3.6.1.2.1.15.3.1.2.10.0.0.1|2|6
+1.3.6.1.2.1.15.3.1.3.10.0.0.1|2|2
+1.3.6.1.2.1.15.3.1.5.10.0.0.1|64x|0a000002
+1.3.6.1.2.1.15.3.1.7.10.0.0.1|64x|0a000002
```

Comment:
Should these have the same value? or does it not matter?

Reply:
For these tests it does not matter